packages/base/src/dist/Utils.ts

```diff
@@ -0,0 +1,3 @@
+import { deprecationNotice, enrichEventWithDetails } from '../utils';
```

Comment:
I think we don't have a dist folder for this release?

Reply:
Done